packages/shorebird_cli/lib/src/commands/build/build_command.dart

```diff
@@ -18,7 +18,7 @@ class BuildCommand extends ShorebirdCommand {
   String get description => 'Build a new release of your application.';
```

Comment:
Might want to explain what it's for, or that it does not save the release build for later patching, etc.

Reply:
Done